GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -383,30 +409,25 @@ extension Glia {
             switch event {
             case .screenShareOffer(answer: let answer):
                 self?.environment.coreSdk.requestEngagedOperator { operators, _ in
-                    self?.callVisualizer.offerScreenShare(
-                        from: operators ?? [],
-                        configuration: Theme().alertConfiguration.screenShareOffer,
+                    self?.operatorRequestHandlerService.offerScreenShare(
+                        from: operators?.compactMap { $0.name }.joined() ?? "",
                         accepted: { [weak self] in
-                            self?.loggerPhase.logger.prefixed(Self.self).info("Screen sharing accepted by visitor")
-                            answer(true)
+                            self?.callVisualizer.observeScreenSharingHandlerState()
```

Comment:
don't we need logs anymore here?

Reply:
We log this even within OperatorRequestHandlerService